data/sources.csv

```diff
@@ -0,0 +1,192 @@
+data_type	short_source	source
```

Comment:
Is this a "CSV" when it's tab separated? :)

Reply:
what do you prefer? `sources.conf` instead?

Reply:
changed to `sources.config`